packages/E2ETest/wdio/test/Image.spec.ts

```diff
@@ -27,7 +27,7 @@ describe('ImageWithoutBorderTest', () => {
   });
 
   /* Test case #3: Click button one more, remove border from view and image but tree sturcture is different from #1*/
-  it('ImageWithoutBorderTest', () => {
+  it('ImageWithoutBorderTestOneMoreClick', () => {
```

Comment:
@dannyvv  this maps to `ImageWithoutBorder-Subsequent.json` - can you name this the same?